liquigraph-core/src/main/java/org/liquigraph/core/api/Liquigraph.java

```diff
@@ -70,8 +69,8 @@ private static ChangelogDiffMaker changelogDiffMaker() {
         return new ChangelogDiffMaker();
     }
 
-    private static ChangelogGraphReader changelogGraphReader() {
-        return new ChangelogGraphReader();
+    private static ChangelogGraphReaderImpl changelogGraphReader() {
```

Comment:
per our previous discussion (about implementing `clearChecksums` as a private method in `ChangelogGraphReader`, this change won't be necessary

Reply:
done in commit eee1086c

But I needed to change ChangelogGraphReader with a Function<Collection<Changeset>, ChangelogGraphReader> since the graphReader now needs the declaredChangeSet.